client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -86,6 +86,7 @@ object CapiModelEnrichment {
         tagExistsWithId("tone/analysis") -> AnalysisDesign,
         tagExistsWithId("tone/comment") -> CommentDesign,
         tagExistsWithId("tone/letters") -> LetterDesign,
+        tagExistsWithId("theguardian/series/correctionsandclarifications") -> CorrectionDesign,
```

Comment:
Quick check on this: any chance we'll ever have this as well as other design flags e.g. letter or comment? If so it'll need to go higher up and have a test to check the edge cases

Reply:
From what I've seen, there aren't any other flags for this.